tests/make-nohistory-image

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+#
+# make-no-history-image - craft a custom image used in bud-implicit-no-history test
+#
+# See https://github.com/containers/buildah/pull/5473/files
+#
+set -e
+
+# Name of the resulting image. (Never pushed automatically).
+# If image build is successful, a human can push it to quay.
+IMAGE=quay.io/libpod/buildah-testimage-nohistory:$(date +%Y%m%d)
+
+function createrandom() {
+    outfile=${1?Missing OUTFILE arg to createrandom()}
+    if [[ ! -e $outfile ]]; then
+        dd if=/dev/urandom bs=1 count=${2:-256} of=$outfile status=none
+    fi
+}
+
+workdir=$(mktemp -d)
+
+# First layer is a copy of this script
+# FIXME: find some way to add a label or annotation or something to indicate that
+cp $0 $workdir/file01
```

Comment:
This should probably be "$workdir/file1", since seq isn't zero-padding the values it produces.

Reply:
It is now :-)

Reply:
I don't understand what you mean here.

Reply:
Oh..... duh, my bad. I changed `seq` to `seq --format %02g` ... but apparently I need to actually commit that change.